.moon/tasks/go.yml

```diff
@@ -48,3 +50,4 @@ tasks:
     options:
       envFile:
         - "/dev.env"
+      outputStyle: "stream"
```

Comment:
this is automatic when the task name is `dev` and when `local: true`: https://moonrepo.dev/docs/config/project#local